src/Sylius/Bundle/AttributeBundle/Resources/views/attributeValueForms.html.twig

```diff
@@ -14,9 +14,9 @@
             <div class="form-group">
                 {% set id = form.vars.label|replace(' ', '_')|lower %}
                 <label class="col-lg-2 control-label required" for="{{ id }}">{{ form.vars.label }}</label>
-                <input type="hidden" name="sylius_product[attributes][{{ count }}][attribute]" id="sylius_product_attributes_{{ count }}_attribute" value="{{ attributeId }}"/>
+                <input type="hidden" name="sylius_{{ subject }}[attributes][{{ count }}][attribute]" id="sylius_{{ subject }}_attributes_{{ count }}_attribute" value="{{ attributeId }}"/>
```

Comment:
I think it was a talk about attributes collection name some time ago, but it has [not been fixed yet](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Product/Model/Product.php#L56). Sure it should be, but I think it's not a case of this PR ;)